pkg/kubelet/runtime_hooks.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubelet
+
+import (
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/client/record"
+	kubecontainer "github.com/GoogleCloudPlatform/kubernetes/pkg/kubelet/container"
+	"github.com/golang/glog"
+)
+
+// Kubelet-specific runtime hooks.
+type kubeletRuntimeHooks struct {
+	recorder record.EventRecorder
+}
+
+var _ kubecontainer.RuntimeHooks = &kubeletRuntimeHooks{}
+
+func newKubeletRuntimeHooks(recorder record.EventRecorder) kubecontainer.RuntimeHooks {
+	return &kubeletRuntimeHooks{
+		recorder: recorder,
+	}
+}
+
+func (kr *kubeletRuntimeHooks) ShouldPullImage(pod *api.Pod, container *api.Container, imagePresent bool) bool {
+	if container.ImagePullPolicy == api.PullNever {
```

Comment:
I'm not sure I understand what you mean. Do you mean having a single condition (or less than we have today)? I thought going this route was more readable since the condition below was already long. Or maybe I didn't understand your comment?